src/reynir/verbframe.py

```diff
@@ -353,6 +359,11 @@ def matches_arguments(cls, verb_with_cases: str) -> bool:
             with its configured arguments? """
         return verb_with_cases in cls.CASE_FRAMES
 
+    @classmethod
+    def matches_error_arguments(cls, verb_with_cases: str) -> bool:
+        """ Does the given key, match any erroneous verb frames? """
+        return verb_with_cases in cls.WRONG_CASE_FRAMES
```

Comment:
This looks suspicious - cls.WRONG_CASE_FRAMES is not a list of strings, it's a list of VerbFrame instances.

Reply:
The key in cls.WRONG_CASE_FRAMES is a string, this checks if verb_with_cases is a key. Changed to cls.WRONG_CASE_FRAMES.keys() to make this clearer